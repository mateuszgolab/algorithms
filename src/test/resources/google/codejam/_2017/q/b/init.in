22
132
1000
7
111111111111111110
123471
100100
123456709
9
10
1000000000000000000
100002341118
2340000823823
111009
112002320
10101
12340000
11110
987
9876543210
11111111111111111
123450001
111223333334444406