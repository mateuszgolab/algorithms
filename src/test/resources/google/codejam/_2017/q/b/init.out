Case #1: 129
Case #2: 999
Case #3: 7
Case #4: 99999999999999999
Case #5: 123469
Case #6: 99999
Case #7: 123456699
Case #8: 9
Case #9: 9
Case #10: 999999999999999999
Case #11: 99999999999
Case #12: 2339999999999
Case #13: 99999
Case #14: 111999999
Case #15: 9999
Case #16: 12339999
Case #17: 9999
Case #18: 899
Case #19: 8999999999
Case #20: 11111111111111111
Case #21: 123449999
Case #22: 111223333333999999
